db/comdb2.c

```diff
@@ -6740,6 +6754,7 @@ int main(int argc, char **argv)
     if (comdb2ma_stats_cron() != 0)
         abort();
 
+    local_rep_sched();
```

Comment:
Is it possible to call it only if there's a local replicant (to reduce the number of threads)?